src/main/java/firstSimpleJavaTasks/ArrayGenerating.java

```diff
@@ -0,0 +1,26 @@
+package firstSimpleJavaTasks;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ArrayGenerating {
+
+    //method for array generating and filling it by random int numbers from 0 to 300 using List array.
+    static List<Integer> generateRandomArrayList (int arrayLength) {
+        List<Integer> newArrayList = new ArrayList<>();
+        for (int i = 0; i < arrayLength; i++) {
+            newArrayList.add(i, (int) (Math.random() * 300));
```

Comment:
`300` good candidate to extract into class constant

Reply:
New constant created